_episodes/09-conflict.md

```diff
@@ -97,21 +98,22 @@ $ cat mars.txt
 Cold and dry, but everything is my favorite color
 The two moons may be a problem for Wolfman
 But the Mummy will appreciate the lack of humidity
-We added a different line in the other copy
+A tall, glass base would be best, to give Vlad the best view of the red surroundings
+A helipad could be useful for supplies arriving
 ~~~
 {: .output}
 
 We can commit the change locally:
 
 ~~~
 $ git add mars.txt
-$ git commit -m "Add a line in my copy"
+$ git commit -m "Add points around base layout"
```

Comment:
Again, something in the story but is easy for a novice learner to see that this line was added from Vlad's copy. Maybe

> Vlad's suggestions on base layout

Reply:
This would subvert the intent of this pull request. The point is to demonstrate good git commit practice as much as possible, since excluding exercises, in the full lesson a learner will only see an instructor make 9 commits. If two of those include data around who authored the commit (which shouldn't be in a commit message, because that data is recorded separately by Git—as we discuss in episode 4), then there is a risk that novices will pattern their commit messages on this style and include redundant information in their commit messages. This risk is elevated because these are the last two commits that a learner will see an instructor make in this lesson.

I"m not sure that having difficulty following who is playing who, or whose commits are whose, in this section is specific to being a novice Git user. Would it be better to add extra expository text before the commit (line 104/106 in the diff above) to reaffirm that this is specific to Vlad/the repository owner's copy.